src/renderers/webgl/WebGLCapabilities.js

```diff
@@ -22,6 +24,33 @@ function WebGLCapabilities( gl, extensions, parameters ) {
 
 	}
 
+	function pixelFormatReadable( textureFormat ) {
```

Comment:
I would prefer to keep `textureFormatReadable()`. It's the more consistent naming.